docs/docs/getting-started.md

```diff
@@ -106,11 +106,11 @@ For Web, you'll additionally need to add the MapLibre CSS to your page. The
 easiest way to do this is via the CDN:
 
 ```kotlin title="index.html"
-<!doctype html>
-<html lang="en">
-  <head>
-    <link rel='stylesheet' href='https://unpkg.com/maplibre-gl@{{ gradle.maplibre_js_version }}/dist/maplibre-gl.css'/>
-  </head>
+<!doctype html >
+<html lang ="en">
+<head >
+<link rel ='stylesheet' href = 'https://unpkg.com/maplibre-gl@{{ gradle.maplibre_js_version }}/dist/maplibre-gl.css'/>
```

Comment:
I think the spaces around the `href = ` have to be removed

Reply:
ah nice catch, it must have auto-formatted with the wrong language